nais/dev-sbs/serviceRedirects/veienTilArbeid.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: veientilarbeid
+  namespace: personbruker
+spec:
+  type: ExternalName
+  externalName: veientilarbeid.q1.svc.nais.local
```

Comment:
Vi kan eventuelt sette denne til namespace-et `paw` i stede for `q1`. Hva tenker du @sasoria ? 🙂

Reply:
Jepp, det kan være lurt å bruke paw her.

Reply:
Nå har jeg byttet til å peke mot paw i dev, og der fungerer fint. Har også oppdatert denne PR-en.